nessie/src/test/java/org/apache/iceberg/nessie/TestNamespace.java

```diff
@@ -62,10 +72,35 @@ public void testListNamespaces() {
   }
 
   @Test
-  public void testDroppingNamespace() {
-    Assertions.assertThatThrownBy(() -> catalog.dropNamespace(Namespace.of("test")))
-        .isInstanceOf(UnsupportedOperationException.class)
-        .hasMessage("Cannot drop namespace 'test': dropNamespace is not supported by the NessieCatalog");
+  public void testCreatingAndDroppingNamespace() {
```

Comment:
additional Namespace testing will be added once we introduce a NessieCatalogTest

Reply:
https://github.com/apache/iceberg/pull/4392